be/src/runtime/runtime_state.cpp

```diff
@@ -414,7 +414,8 @@ Status RuntimeState::append_error_msg_to_file(std::function<std::string()> line,
         }
     }
 
-    if (out.size() > 0) {
+    size_t error_row_size = out.size();
+    if (error_row_size > 0 && error_row_size <= config::load_error_log_limit_bytes) {
```

Comment:
Maybe we should notify the user that the error log is truncated by appending a line at the end of the error log file saying 'truncation: error log is too long'?